extensions/parquet/table/src/test/java/io/deephaven/parquet/table/ParquetTableReadWriteTest.java

```diff
@@ -2650,4 +2652,36 @@ private void assertBigIntegerColumnStatistics(SerialObjectColumnIterator<BigInte
         }
     }
     // endregion Column Statistics Assertions
+
+    @Test
+    public void generateDateTimeCol() {
+        final int NUM_ROWS = 5;
+
+        final int ARRAY_SIZE = 3;
+        final LocalDate[] dateArr = new LocalDate[ARRAY_SIZE];
+        // final LocalTime[] timeArr = new LocalTime[ARRAY_SIZE];
+        for (int ii = 0; ii < ARRAY_SIZE; ++ii) {
+            dateArr[ii] = LocalDate.now();
+            // timeArr[ii] = LocalTime.now();
+        }
+        QueryScope.addParam("dateArr", dateArr);
+        // QueryScope.addParam("timeArr", timeArr);
+        final Table table = TableTools.emptyTable(NUM_ROWS).view(
+                "Date = java.time.LocalDate.now()",
+                // "nullDate = (java.time.LocalDate)null"
+                "DateArray = dateArr");
+        final File dest = new File("/Users/shivammalhotra/Documents/dates.parquet"); // TOD change it back to local dir
```

Comment:
This is clearly not ready to merge as committed. Please be sure to fix both test file paths, and test coverage for the new/changed code.